src/radio/radio-controller.ts

```diff
@@ -99,12 +102,10 @@ export class RadioController {
     // Timeout if there hasn't been a response.
     return Promise.race([
       discoveryBuffer,
-      this.createTimeoutPromise().then(() => {
+      this.createTimeoutPromise(timeout).then(() => {
         // Close the socket if timed out
         socket.close();
-        throw new Error(
-          'UDP scan timed out after ' + RADIO_TIMEOUT.toString() + 'ms.'
-        );
+        throw new Error(`UDP scan timed out after ${RADIO_TIMEOUT}ms.`);
```

Comment:
If you're using `timeout` as a param then we should put that in the error message.

Reply:
Done